.github/workflows/reusable-helmfile-deploy.yaml

```
@@ -0,0 +1,44 @@
+---
+name: Deploy Helmfile
+on:
+  workflow_call:
+    inputs:
+      ENVIRONMENT:
+        required: false
+        type: string
+        default: ""
+      WORKDIR:
+        required: false
+        type: string
+        default: ./
+      HELMFILE_NAME:
+        required: false
```

```suggestion
        required: true
```

`HELMFILE_NAME` should be required, right?